.github/workflows/ci.yaml

```diff
@@ -30,6 +30,13 @@ jobs:
           {"folder": ".", "bzlmodEnabled": true}
         ]
 
+  repository-rule-deps:
```

Comment:
this doesn't need a new job, you can just add to the `folders` array of the `test` job above